.github/workflows/code-health.yml

```diff
@@ -16,22 +16,34 @@ jobs:
     runs-on: ubuntu-latest
     steps:
```

Comment:
should we add a small docs section in our CONTRIBUTING file for when and how mockery should be used? Similar to https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/CONTRIBUTING.md#code-and-test-best-practices.
We can specify the convention of defining in `internal/testutil/mocksvc/*.go`.

Reply:
good idea, I'll add it

Reply:
done